src/bp/ui-studio/src/web/views/FlowBuilder/diagram/nodes_v2/style.scss

```diff
@@ -52,29 +73,29 @@
 
 .nodeSaySomething {
   .header {
-    background: #a1c6e8;
-    color: #ffffff;
+    background: #D1E5F2;
+    color: #182026;
   }
 }
 
 .nodeExecute {
   .header {
     background: #d5a5bc;
-    color: #ffffff;
+    color: #182026;
   }
 }
 
 .nodeListen {
   .header {
     background: #ffe699;
-    color: #000000;
+    color: #182026;
   }
 }
 
 .nodeRouter {
   .header {
     background: #a1c4c9;
-    color: #000000;
+    color: #182026;
```

Comment:
You could extract #182026 to a constant and reuse it

Reply:
Yeah I'm looking into doing that in my new branch, both will be merged into a feature branch so I'll leave it as is for this branch and replace them once I merge my branches together